setup.py

```diff
@@ -1,14 +1,13 @@
-#from distutils.core import setup
 from os import path
-import setuptools
+from setuptools import setup
 here = path.abspath(path.dirname(__file__))
 
-version_num = "1.2.2"
+version_num = "1.2.3"
```

Comment:
Not sure how you wish to handle versioning.

Reply:
adding python 3 support is prettty big. might as well make it 2.0.0.